multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -160,7 +169,12 @@
     BINARY_ACC: MAX,
     BER: MIN,
 }
-VALID_METRICS = METRIC_MODE_MAP.keys()
+
+MATCHING_METRICS = {
+    MULTICLASS: [SPEARMANR, SPEARMANR],
+    REGRESSION: [SPEARMANR, SPEARMANR],
+}
+MATCHING_METRICS_WITHOUT_PROBLEM_TYPE = [RECALL, NDCG]
```

Comment:
recall should be the validation metric, but used as eval metric in `infer_metrics`.